### PR TITLE
fix(*): update codegen and formatting commands

### PR DESCRIPTION
**Pull request by BhavyaRattan**
### Description:

Update commands for codegen and formatting

### CODE REVIEW

File: lib/fastlane/plugin/fueled/actions/formatting_checks_flutter.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class FormattingChecksFlutterAction < Action
       def self.run(params)
         UI.message("Checking formatting")
         sh("flutter pub get")
-        sh("dart format -o none --set-exit-if-changed .")
+        sh("dart format $(find lib test integration_test -name \"*.dart\" -not \( -name \"*.*freezed.dart\" -o -name \"*.g.dart\" -o -name \"*.gen.dart\" -o -name \"*.mocks.dart\" \) ) -o none --set-exit-if-changed")
```
Comment (BhavyaRattan) [inline lib/fastlane/plugin/fueled/actions/formatting_checks_flutter.rb:10]: This change was needed as dart format does not exclude the files specified in analysis options. [Reference](https://github.com/dart-lang/dart_style/issues/864#issuecomment-1233176900)

**Review by shkcodes — APPROVED** [review]

File: lib/fastlane/plugin/fueled/actions/formatting_checks_flutter.rb (hunk 1 above)
Comment (humblerookie) [reply lib/fastlane/plugin/fueled/actions/formatting_checks_flutter.rb:10]: can we pull the internal command into a local variable for readability.

**Review by thapli9A — APPROVED** [review]
LGTM

File: lib/fastlane/plugin/fueled/actions/formatting_checks_flutter.rb (hunk 1 above)
Comment (BhavyaRattan) [reply lib/fastlane/plugin/fueled/actions/formatting_checks_flutter.rb:10]: done!

**Pull request merged**